pkg/cosign/remote/index.go

```diff
@@ -141,6 +145,9 @@ func UploadFiles(ref name.Reference, files []File, getMt MediaTypeGetter, remote
 	}
 
 	if len(files) > 1 {
+		if annotations != nil {
+			idx = mutate.Annotations(idx, annotations).(v1.ImageIndex)
```

Comment:
If we're already including annotations when the file is initially created in line 112, do we need to add them in again here?